lib/workers/branch/handle-existing.ts

```diff
@@ -3,8 +3,11 @@ import { logger } from '../../logger';
 import { Pr, platform } from '../../platform';
 import { PrState } from '../../types';
 import { branchExists, deleteBranch } from '../../util/git';
+import { RepositoryStatisticsReporter } from '../../util/stats-reporter';
 import * as template from '../../util/template';
 import { BranchConfig } from '../types';
+//import { StatsReporter } from '../../util/stats-reporter';
+//import { DependencyStats } from '../../util/stats-reporter/types';
```

Comment:
?

Reply:
There has been some iterations on the code and I forgot to clean that up. It's gone now though. Thanks